src/render/pic.rs

```diff
@@ -0,0 +1,72 @@
+//! PIC rendering support.
+//!
+//! # Example
+//!
+//! ```
+//! use qrcode::QrCode;
+//! use qrcode::render::pic;
+//!
+//! let code = QrCode::new(b"Hello").unwrap();
+//! let pic = code.render::<pic::Color>().build();
+//! println!("{}", pic);
+
+#![cfg(feature = "pic")]
+
+use std::fmt::Write;
+use std::marker::PhantomData;
+
+use crate::render::{Canvas as RenderCanvas, Pixel};
+use crate::types::Color as ModuleColor;
+
+/// A PIC color.
+#[derive(Copy, Clone, Default, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct Color;
+
+impl Pixel for Color {
+    type Canvas = Canvas;
+    type Image = String;
+
+    fn default_color(_color: ModuleColor) -> Self {
+        Color
+    }
+}
+
+#[doc(hidden)]
+pub struct Canvas {
+    pic: String,
+    marker: PhantomData<Color>,
```

Comment:
```suggestion
```

no need to include the PhantomData since it no longer needs to capture the unused generic argument.